app/layout/community/published-projects/duplicate-button/component.tsx

```diff
@@ -18,32 +19,62 @@ export const DuplicateButton: React.FC<DuplicateButtonProps> = ({
 }: DuplicateButtonProps) => {
   const { addToast } = useToasts();
 
-  const handleDuplicated = () => {
-    addToast(`success-project-duplicated-${id}`, (
-      <>
-        <h2 className="font-medium">Success!</h2>
-        <p className="text-sm">
-          Project
-          {' '}
-          {name}
-          {' '}
-          duplicated
-        </p>
-      </>
-    ), {
-      level: 'success',
+  const duplicateProjectMutation = useDuplicateProject({
+    requestConfig: {
+      method: 'POST',
+    },
+  });
+
+  const onDuplicate = useCallback(() => {
+    // Name must be the new one defined by the user
+    duplicateProjectMutation.mutate({ id, data: name }, {
```

Comment:
`data` is not part of the props, that's why the deploy is failling. For now, don't add it.

Reply:
resolved it.